datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/util/ActionableFileUtil.java

```diff
@@ -162,6 +166,8 @@ public static String toLine(@NotNull ActionableEvent event) {
                 .add(setToField(event.sourceUrls()))
                 .add(clinicalTrial != null ? clinicalTrial.studyNctId() : Strings.EMPTY)
                 .add(clinicalTrial != null ? clinicalTrial.studyTitle() : Strings.EMPTY)
+                .add(clinicalTrial != null ? clinicalTrial.studyAcronym() : Strings.EMPTY)
+                .add(clinicalTrial != null ? clinicalTrial.gender() : Strings.EMPTY)
```

Comment:
Since the new fields are nullable, should they be set to null when no trial? I'm not sure what the conventions are.

Reply:
Reason is that nullable values should be present as empty strings in the actionable output files

Reply:
Oh this was file writing, yes thanks!